api/src/main/java/com/thesurvey/api/exception/GlobalAPIExceptionHandler.java

```diff
@@ -1,16 +1,65 @@
 package com.thesurvey.api.exception;
 
+import javax.validation.ConstraintViolationException;
+import javax.validation.UnexpectedTypeException;
+import org.postgresql.util.PSQLException;
 import org.springframework.http.HttpStatus;
 import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.MethodArgumentNotValidException;
 import org.springframework.web.bind.annotation.ExceptionHandler;
 import org.springframework.web.bind.annotation.RestControllerAdvice;
 
 @RestControllerAdvice
 public class GlobalAPIExceptionHandler {
 
-    @ExceptionHandler
-    public ResponseEntity<String> handleNotFoundException(ExceptionMapper e) {
-        return ResponseEntity.status(HttpStatus.NOT_FOUND).body(e.getMessage());
+    /*
+        @handleExceptionMapperException handles the exception thrown by ExceptionMapper.
+    */
```

Comment:
It doesn't apply to JavaDoc. Please modify this to 

```java
/**
 * Handles exception thrown by ExceptionMapper.
 */
```

And please I'm repeating this: **Follow code conventions**.